src/resources/svg/no-transactions.svg

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This file is SVG  that can be used in other places. What do you think about renaming the file as `empty.svg`?